### PR TITLE
Add format=json parameter to get-availability-zone

### PR DESCRIPTION
**Pull request by r-f-g**
- add paramter format=json to get-availability-zone action
 - change custom charm to openstack-charmers-next

fixes: [#1944510](https://bugs.launchpad.net/juju-verify/+bug/1944510)

### CODE REVIEW

**Review by auria — APPROVED** [review]
LGTM.

**Review by auria — APPROVED** [review]
LGTM after rebase

**Review by r-f-g — COMMENTED** [review]
recheck-full

**Review by r-f-g — COMMENTED** [review]
recheck-full

File: README.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ Juju-verify requires Juju 2.8.10 or higher.
 ## Supported charms
 
 * nova-compute (Usable with the next stable release of the charm. Currently available as a [nova-compute-rc])
-* ceph-osd (Usable with the custom release of the charm in rgildein namespace. [cs:~openstack-charmers-next/ceph-osd] and [cs:~openstack-charmers-next/ceph-mon])
-* ceph-mon (Usable with the custom release of the charm in rgildein namespace. [cs:~openstack-charmers-next/ceph-mon])
+* ceph-osd (Usable with the next stable of the charm. [cs:~openstack-charmers-next/ceph-osd] and [cs:~openstack-charmers-next/ceph-mon])
```
Comment (auria) [inline README.md:14]: "next stable release of the charm". :)

**Review by auria — CHANGES_REQUESTED** [review]
Minor fix, otherwise +1.

**Review by auria — APPROVED** [review]
+1

**Pull request merged**